src/components/containers/ChartColors.js

```diff
@@ -0,0 +1,11 @@
+// See https://sashat.me/2017/01/11/list-of-20-simple-distinct-colors/
+export default [
+    { r: 74, g: 144, b: 226 },
+    { r: 230, g: 25, b: 75 },
+    { r: 60, g: 180, b: 75 },
+    { r: 0, g: 128, b: 128 },
+    { r: 245, g: 130, b: 49 },
+    { r: 145, g: 30, b: 180 },
+    { r: 70, g: 240, b: 240 },
+    { r: 250, g: 190, b: 190 },
+];
```

Comment:
I wonder if [src/lib](https://github.com/mozilla/firefox-test-tube/tree/master/src/lib) might be a better place for this file. So far, everything in *containers/* is a React container component.

Reply:
yea good catch - only here temporarily since i wasn't settled on the approach to share the color definitions